PURL-TYPES.rst

```diff
@@ -515,21 +515,22 @@ pypi
 ----
 ``pypi`` for Python packages:
 
-- The default repository is ``https://pypi.org``. (Previously  ``https://pypi.python.org``.)
-- PyPI treats ``-`` and ``_`` as the same character and is not case sensitive.
-  Therefore a PyPI package ``name`` must be lowercased and underscore ``_``
-  replaced with a dash ``-``.
+- The default repository is ``https://pypi.org``. (Previously ``https://pypi.python.org``.)
+- Per `PEP 503 <https://peps.python.org/pep-0503/#normalized-names>`_, normalized PyPI package names should be 
+  lowercased with all runs of the characters ``.``, ``-``, or ``_`` replaced with a single ``-`` character.
```

Comment:
The correct document is https://packaging.python.org/en/latest/specifications/name-normalization/, not the PEP.